src/core/apiLayer/productService.ts

```diff
@@ -0,0 +1,125 @@
+import { stringify } from "query-string";
+import { useEffect, useReducer } from "react";
+
+import { apiUrl } from "@temp/constants";
+
+const fetchProducts = (criteria: any, limit: number, offset: number) => {
+  const query = stringify({
+    ...criteria,
+    currency: "EUR",
+    limit,
+    offset,
+  });
+
+  return fetch(`${apiUrl}../pkb/products/?${query}`, {
```

Comment:
what is `..`? :)

Reply:
Yes, need to create a env vars for this, also for the auth header. :)

Reply:
I mean what `..` does in the template literal, just curious.

Reply:
Ow 😂 it just move one folder up. apiUrl is https://host/graphql/ this makes it https://host/pkb/products/

Reply:
Ah, is it `fetch`'s functionality?

Reply:
Yes/no it is browser functionality. In the address bar of your browser you can write: https://google.com/hoi/../search it will be translated to https://google.com/search

Reply:
Works same as `cd ..`